Libs/MRML/DisplayableManager/vtkMRMLModelDisplayableManager.cxx

```diff
@@ -1049,7 +1049,6 @@ void vtkMRMLModelDisplayableManager::UpdateModelMesh(vtkMRMLDisplayableNode *dis
     vtkProp3D* prop = nullptr;
 
     int clipping = displayNode->GetClipping();
-    int visibility = (displayNode->GetVisibility() == 1 && displayNode->GetVisibility3D() == 1 ? 1 : 0);
```

Comment:
Yes, it is. The block

```
      if (modelDisplayNode)
        {
        this->Internal->DisplayedVisibility[modelDisplayNode->GetID()] = visibility;
        }
      else
        {
        this->Internal->DisplayedVisibility[modelDisplayNode->GetID()] = 1;
        }
```

has been removed because the DisplayedVisibility map has been also removed.

Reply:
Thanks for checking :pray: